www/html/ci/application/views/landing.php

```diff
@@ -1,11 +1,86 @@
 {% extends "base.html" %}
 
-{% block extra %}<?php
+
+{% block extra %}
+<?php
 defined('BASEPATH') OR exit('No direct script access allowed');
 ?>
 {% endblock %}
+{% block head %}
+
+
+	<meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
+	<title>Ken Torii &copy;</title>
+	<meta content="width=device-width, initial-scale=1" name="viewport" />
+	<link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
+	<link href="{{ base_url }}font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
+	<link href="{{ base_url }}css/index2.css" rel="stylesheet" />
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+	<script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+	<script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+	<script type="text/javascript" src="{{ base_url }}js/webJS.js"></script>
+
+{% endblock %}
+
+{% block content1 %}
+
+<!-- Modal -->
+<div id="myModal" class="modal fade" role="dialog">
+	<div class="modal-dialog">
+
+		<!-- Modal content-->
+		<div class="modal-content">
+			<div class="modal-header">
+				<button type="button" class="close" data-dismiss="modal">&times;</button>
+				<h4 class="modal-title">Login</h4>
+			</div>
+			<div class="modal-body">
+
+				<div id='errorMessage'>
+		        {{error|nl2br}}
+		    </div>
+
+		    <form method="POST" action="controllers/Landing.php" name="signIn" id="signInForm">
+
+		        <h5>Username</h5>
+		        <input class="form-control" type="text" name="username" value="" size="50" /><br>
+
+		        <h5>Password</h5>
+		        <input class="form-control" type="password" name="password" value="" size="50" />
+
+		        <div><input class="btn" type="submit" value="Login" /></div>
+
+		    </form>
+
+		    <script type='text/javascript'>
```

Comment:
javascript should be in seperate file. This is russell's function right. Where was it before. If youc ant' call it from there, put it in the webjs file